Tests/ImageTintingTests.swift

```diff
@@ -85,4 +85,13 @@ class ImageTintingTests: XCTestCase {
         BONAssertEqualImages(untintedResult!, tintAttemptResult!)
     }
 
+    func testAccessibilityLabelTransferToTintedImage() {
+        let accessibilityLabel = "I’m the very model of a modern accessible image."
+        let accessibleImage = UIImage(cgImage: imageForTest.cgImage!)
```

Comment:
Looked into it on WatchKit a bit and couldn’t come up with anything. I don’t have any Apple Watch apps at the moment. Will look into it again when I can and possibly file a radar. Do you know of anyone using the framework in an open source Watch app I can look at?